### PR TITLE
Fixes after release

### PR DESCRIPTION
**Pull request by moxiegirl**
Adding in a link about the deprecated material
Fixing broken images in install
Fixing some broken links in trust
Adding in some build facilities for making test eaiser

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by jfrazelle** [comment]
LGTM

**Comment by moxiegirl** [comment]
author LGTM

File: docs/reference/api/hub_registry_spec.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+keywords = ["docker, registry, api,  hub"]
+[menu.main]
+parent="smn_hub_ref"
++++
+<![end-metadata]-->
+
+# The Docker Hub and the Registry v1
+
+This API is deprecated as of 1.7. To view the old version, see the [go
+here](http://docs-stage.docker.com/v1.7/reference/api/hub_registry_spec/) in
```
Comment (icecrime) [inline docs/reference/api/hub_registry_spec.md]: Should the link be pointing to `docs-stage`?

**Comment by icecrime** [comment]
I'll trust you with the Dockerfile and bash scripts, otherwise LGTM.

**Pull request merged**